sdk/maps/Azure.ResourceManager.Maps/src/autorest.md

```diff
@@ -59,6 +60,8 @@ rename-rules:
   SSO: Sso
   URI: Uri
   Etag: ETag|etag
+  CorsRules: MapsCorsRule
```

Comment:
This needs to be added to `rename-mapping` section in L18, the latest generated code still remains `CorsRule`

Reply:
Oh, sorry, my fault....

Reply:
Adding the line to `rename-mapping` doesn't work? As I leave the message in another comment, I should still add it in `prepend-rp-prefix`?